com.endlessm.CompanionAppService.json.in

```diff
@@ -24,6 +24,27 @@
     ],
     "modules": [
         "com.endlessm.CompanionAppService.PipDependencies.json",
+        {
+            "name": "mustache-c",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/x86-64/mustache-c"
```

Comment:
Ah, I forgot to mention that you'll need to pin this to a certain version. See https://github.com/endlessm/endless-sdk-flatpak/blob/master/com.endlessm.apps.Sdk.json.in.in#L511

Reply:
Done.